src/crimson/net/Socket.h

```diff
@@ -7,8 +7,10 @@
 #include <seastar/core/sharded.hh>
 #include <seastar/net/packet.hh>
 
+#include "common/likely.h"
```

Comment:
is `likely()` used anywhere?

Reply:
Sorry, forgot to remove the unused header from the last version.